### PR TITLE
Ecosystem: added s3 landing to local configuration

### PR DESCRIPTION
**Pull request by nick-otter**

### CODE REVIEW

File: ecosystem.config.js [hunk 1]
```diff
@@ -77,5 +77,23 @@ module.exports = {
         WRITE_AWS_S3_BUCKET_NAME : process.argv[7]
       }
     },
+    {
+      name      : "s3-landing-to-local",
+      script    : "/appdata/data-transfer/bin/data-transfer",
+      interpreter: "python",
+      env: {
+        INGEST_DEST_PATH : "/home/wherescape/test_for_data_transfer_ingest_landing",
```
Comment (david-wirelab) [inline ecosystem.config.js:85]: `test_for_*` why is it named lake that? Can it not be `data_transfer_ingest_landing` instead?

File: ecosystem.config.js [hunk 2]
```diff
@@ -77,5 +77,23 @@ module.exports = {
...
+        INGEST_SOURCE_PATH : "oag/tmp",
+        WRITE_STORAGE_TYPE : "datatransfer.storage.FolderStorage",
+        READ_STORAGE_TYPE : "datatransfer.storage.S3Storage",
+        LOG_FILE_NAME : "s3-landing-to-local.log",
+        PROCESS_INTERVAL : 4,
+        MAX_FILES_BATCH : 25,
+        FOLDER_DATE_OUTPUT : "False",
+        LOG_LEVEL : "DEBUG",
+        USE_IAM_CREDS : "True",
+        READ_AWS_S3_BUCKET_NAME : < landing bucket environment variable goes here >
```
Comment (david-wirelab) [inline ecosystem.config.js]: Should this not be empty?

File: ecosystem.config.js [hunk 3]
```diff
@@ -77,5 +77,23 @@ module.exports = {
...
+      }
+    },
```
Comment (david-wirelab) [inline ecosystem.config.js]: extra comma

File: ecosystem.config.js (hunk 1 above)
Comment (nick-otter) [reply ecosystem.config.js:85]: this is the first pass to test the configuration, will be switched over to the appropriate directory name when the configuration is switched on in prod

File: ecosystem.config.js (hunk 2 above)
Comment (nick-otter) [reply ecosystem.config.js]: Updated to empty!

File: ecosystem.config.js (hunk 3 above)
Comment (nick-otter) [reply ecosystem.config.js]: actually correct!

File: ecosystem.config.js [hunk 4]
```diff
@@ -77,5 +77,23 @@ module.exports = {
...
+        READ_STORAGE_TYPE : "datatransfer.storage.S3Storage",
+        LOG_FILE_NAME : "s3-landing-to-local.log",
+        PROCESS_INTERVAL : 4,
+        MAX_FILES_BATCH : 25,
+        FOLDER_DATE_OUTPUT : "False",
+        LOG_LEVEL : "DEBUG",
+        USE_IAM_CREDS : "True",
+        READ_AWS_S3_BUCKET_NAME : 
+      }
+    },
```
Comment (david-wirelab) [inline ecosystem.config.js:97]: extra comma

**Review by david-wirelab — CHANGES_REQUESTED** [review]

File: ecosystem.config.js (hunk 4 above)
Comment (nick-otter) [reply ecosystem.config.js:97]: As per line 82 here: https://github.com/UKHomeOffice/data-transfer/blob/master/ecosystem.config.js

**Review by david-wirelab — APPROVED** [review]

**Pull request merged**